alonzo/formal-spec/utxo.tex

```diff
@@ -94,7 +94,25 @@ \subsection{Combining Scripts with Their Inputs}
 \item
   $\fun{indexof}$ is a helper function that finds the index of a given certificate, value, input, or
   withdrawal in a list, finite map, or set of such objects.
-  It assumes there is some ordering on each of these structures.
+  For each of these, it computes the index as follows :
+  \begin{itemize}
```

Comment:
I think we should spell out lexicographical at least once, and abbreviate it as 'lex.' (with the dot)